### DISCUSSION TITLE
I3-Wm desktop

### DISCUSSION THREAD

**Discussion by JesusChapman** [Yisus7u7/termux-desktop-xfce#11] [Ideas]
@suhan-paradkar

Termux has I3-Wm, is it worth making a desktop of it? 
Keep the following in mind :

- The package appears to be incomplete 
-  There is very little software to customize it, you can use polybar, But don't set it up well in termux

**Comment by creepy-pasta101** [comment]
I dont think so... Mate will be available by then... Just mate-control-center has a few problems.. then It will be merged

**Reply by JesusChapman** [reply]
Well, then I'll postpone that, the only way i3 is worth it is that you can put widgets on the screen
